src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { isEmail } from 'validator';
+import { handleForgotPasswordForm, submitForgotPassword } from '../../redux/actions';
+import Input from '../common/Input/Input';
+import Button from '../common/Button/Button';
+
+export class ForgotPassword extends Component {
+  constructor() {
+    super();
+    this.state = {
+      validEmail: true,
+      emailError: 'Email is not valid',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmitButton = this.onSubmitButton.bind(this);
+    this.renderErrors = this.renderErrors.bind(this);
```

Comment:
I'm quite skeptical about this bind statements. Why do you need to add them? If it's an eslint error please send me the error.

Reply:
@kimotho-njoki  The Eslint rules from airbnb that we are using don't allow arrow functions. We are binding `this` in order to have access to props and state within those functions

![image](https://user-images.githubusercontent.com/17350127/55684664-1370ee00-594e-11e9-9376-83d0213510b9.png)

Reply:
Well yes but it still doesn't answer my question. When you fail to bind your statements do you get an error message. Have you tried not binding your statement??

Reply:
Have you tried only having state and eradicating the constructor?

Reply:
I thought we were using Es6... So if our eslint doesn't allow for that why not change the rules??? When using ES5 you will have to keep binding your statements.. Most projects don't do that anymore.. Es6 doesn't need binding.. Research more on that...

Reply:
@kimotho-njoki It is only an Eslint problem. I have changed Eslint rules to allow arrow functions and I also refactored all components accordingly.